docs/server.md

```diff
@@ -0,0 +1,354 @@
+# Neo4j Installation
+1. Download the [Community Server Edition](https://neo4j.com/download-center/#community)
+2. Open up your terminal/shell
+3. Extract the contents of the archive, using: `tar -xf <filecode>`.  For example, `tar -xf neo4j-community-3.5.8-unix.tar.gz`
+4. Place the extracted files in a permanent home on your server. The top level directory is referred to as NEO4J_HOME.
+  - To run Neo4j as a console application, use: `<NEO4J_HOME>/bin/neo4j console`
+  - To run Neo4j in a background process, use: `<NEO4J_HOME>/bin/neo4j start`
+5. Visit [http://localhost:7474](http://localhost:7474) in your web browser.
+6. Connect using the username 'neo4j' with default password 'neo4j'. You'll then be prompted to change the password.
+
+Once you create a new password for the 'neo4j' user upon visiting the Neo4j Browser the first time, you'll have full access to the Neo4j database.
+
+# Neo4j configuration
+1. Create a .env file
+2. Setup the .env file
+    ```
+    // .env
+    NEO4J_PROTOCOL=bolt
+    NEO4J_HOST=localhost
+    NEO4J_USERNAME=neo4j
+    NEO4J_PASSWORD=YOUR_PASSWORD
+    NEO4J_PORT=7687
+    ```
+
+## Getting Started
+1. Make sure your neo4j database is upon running
+2. `npm start`
+
+## Seed database
+`npm run seed`
+
+## Earse database
+`npm run dropDB`
+
+# Resources
+## Certification
+
+### List of all certifications
+`GET /api/certifcation`
+
+Response
+
+Status code: 202
```

Comment:
Why `202` instead of `200`? 

To my interpretation 202 implies that the request has been received but the data is not ready yet.